ale/drivers/lro_driver.py

```diff
@@ -6,17 +6,22 @@
 import spiceypy as spice
 
 from ale.util import get_metakernels
-from ale.drivers.base import LineScanner
-from ale.drivers.distortion import RadialDistortion
+from ale.drivers.base import LineScanner, RadialDistortion, Spice, PDS3, Isis3
 
-class LRO_LROC(LineScanner, RadialDistortion):
+class LrocSpice(Spice, LineScanner, RadialDistortion):
```

Comment:
I like this. Not sure what others think, but having the declaration look like this, I know exactly where the data are from and what the composite parts are that make up this particular driver.

Reply:
For what it's worth, I like this, too.